index.html

```diff
@@ -0,0 +1,95 @@
+            <p class="intro-1">"Hello! Sharing world"</p>
+            <h1>Be Happy, Let's fight for the peace throught Live Music Concert.</h1>
+            <p class="intro-2">A joyfull dayS to celebrate freedom trought music in the East the Republic Democratic of Congo. This activity help to reminde each others that a we can hope for a bright future.</p>
+            <h3>2023.02.23(THU)~25(SAT)</h3>
+            <p class="intro-3">@National Museum of Goma/North Kivu. Art Center and more.</p>
+        </section>
+
+        <section class="programs">
+            <h2>Main Program</h2>
+            <div class="line"></div>
+            <div class="program-container">
+                <div class="program">
+                    <div class="program-icon"><img src="image/lecture-conference-svgrepo-com.svg" alt="lecture"></div>
+                    <h3 class="program-title">Lecture</h3>
+                    <p>Listen to speakers from various tribe and province about the message of peace and love.</p>
+                </div>
+                <div class="program">
+                    <div class="program-icon"><img src="image/exhibition-museum-svgrepo-com.svg" alt="lecture"></div>
+                    <h3 class="program-title">Exhibition</h3>
+                    <p>Listen to speakers from various tribe and province about the message of peace and love.</p>
+                </div>
+                <div class="program">
+                    <div class="program-icon"><img src="image/comment-material-2-svgrepo-com.svg" alt="lecture"></div>
+                    <h3 class="program-title">Forum</h3>
+                    <p>Listen to speakers from various tribe and province about the message of peace and love.</p>
+                </div>
+                <div class="program">
+                    <div class="program-icon"><img src="image/workshop-svgrepo-com.svg" alt="lecture"></div>
+                    <h3 class="program-title">Workshop</h3>
+                    <p>Listen to speakers from various tribe and province about the message of peace and love.</p>
+                </div>
+                <div class="program">
+                    <div class="program-icon"><img src="image/light-svgrepo-com.svg" alt="lecture"></div>
+                    <h3 class="program-title">Ignite</h3>
+                    <p>Listen to speakers from various tribe and province about the message of peace and love.</p>
+                </div>
+            </div>
+
+            <button class="Program-btn" type="button">Join us to 2023 Festival Amani.</button>
+            <p class="view-program"><a href="#">SEE THE WHOLE PROGRAM</a></p>
+        </section>
+
+        <section class="features">
+            <h2>Featured Speakers</h2>
+            <div class="line"></div>
+            <div id="speaker-id"></div>
+
+            <button class="feature-btn" type="button">MORE <img src="image/down-arrow.svg" alt="down-arrow"></button>
+        </section>
+    </main>
```

Comment:
- [ ] I would strongly advise you to follow the Behance design as much as possible as this will show your ability to replicate any given design. Currently, In your design, the `desktop view` of the `Feature speakers` looks different from what we have in the original design, all the `6 feature speaker` cards are displayed in rows and columns without the `MORE` button while on `mobile view`, only `2 feature speaker` cards are displayed with the `MORE` button. Optionally you can implement the more speakers functionality whereby when you click on the button the remaining `4 feature speakers` will be displayed.

Kindly fix this and be sure your design looks like what we have in the original design.

Behance design 👇 

| Desktop | mobile |
| ------------- | ------------- |
| ![beh0](https://user-images.githubusercontent.com/78634841/217484359-b039e0f7-f9be-448c-89ed-7aec535ed620.png)  | ![behmob](https://user-images.githubusercontent.com/78634841/217484464-abd72d3f-c852-4deb-b119-5cb3d252f07a.png)  |